src/highdicom/sr/templates.py

```diff
@@ -3732,6 +3894,225 @@ def __init__(
                 item.RelationshipType = relationship_type.value
                 self.append(item)
 
+    def _generate_projection_radiography_descriptors(
+        self,
+        dataset: Dataset
+    ) -> Sequence[ContentItem]:
+
+        """Generate descriptors for projection radiography modalities.
+        :dcm:`TID 1603 <part16/chapter_A.html#sect_TID_1603>`
+        Image Library Entry Descriptors for Projection Radiography
+
+        Parameters
+        ----------
+        pydicom.Dataset
+            Metadata of a projection radiology image
+
+        Returns
+        -------
+        Sequence[highdicom.sr.ContentItem]
+            SR Content Items describing the image
+
+        """  # noqa: E501
+        patient_orientation = dataset.PatientOrientation
+        pixel_spacing = dataset.ImagerPixelSpacing
+        descriptors = [
+            TextContentItem(
+                name=codes.DCM.PatientOrientationRow,
+                value=patient_orientation[0],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+            ),
+            TextContentItem(
+                name=codes.DCM.PatientOrientationColumn,
+                value=patient_orientation[1],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+            ),
+            NumContentItem(
+                name=codes.DCM.HorizontalPixelSpacing,
+                value=pixel_spacing[1],
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
+                unit=CodedConcept(
```

Comment:
note that the defined UCUM codes are in pydicom too, so a lot of this could be replaced with `codes.UCUM.mm`

Reply:
Yes - the highdicom_requirements.txt is deleted. I've been playing around with the requirements to try to get all of the tests to run on my M1 Mac. There is one test that still fails (TestEncodeFrame.test_jpegls_monochrome) - there is some issue with jpeg-ls component.  But this doesn't happen on Intel Linux machines.

I've replaced the constructors for the UCUM elements with values from pydicom for the most part. The ones with the annotations for {pixels} have been replaced `codes.UCUM.NoUnits`. The only ones with annotations that are left are for the orientation vectors where the values are constrained to `{-1:1}`.

Reply:
The odd {pixels} UCUM annotations were restored because the DicomSRValidator.sh validator requires them.

Reply:
ok I just wanted to check that there was a reason behind this because it seems a little strange to me